_docs_integrate/establish-relationships.md

```diff
@@ -87,7 +87,7 @@ If the RelationshipTemplate is only for creating a Relationship with a single kn
     // RelationshipTemplateContent or ArbitraryRelationshipTemplateContent
     ...
   },
-  "forIdentity": "<address of Identity designated for RelationshipTemplate>"
+  "forIdentity": "<address of Identity the RelationshipTemplate is for>"
```

Comment:
```suggestion
  "forIdentity": "<address of Identity the RelationshipTemplate is intended for>"
```
Currently it sounds like German English :D

Reply:
I wanted to keep it as short as possible, because otherwise a scroll bar would be inserted in the payload box. ;) Do you perhaps have a shorter version that you prefer, or would you rather put up with the scrollbar?

Reply:
What I did for the personalized templates was `address that the RelationshipTemplate is intended for` - and I have no scrollbar :)

Reply:
Strictly speaking, the RelationshipTemplate is not intended for an `address`, but for an Identity, which is why I would like to have "address of Identity" in there in any case. Unfortunately, this leaves less space for the remaining characters. :D